src/app.py

```diff
@@ -49,7 +50,10 @@
 
 @asynccontextmanager
 async def lifespan(app: FastAPI):
-    db.alembic_upgrade()
+    try:
+        db.alembic_upgrade()
+    except Exception:
+        logging.exception("Failed to apply migrations. Trying to continue...")
```

Comment:
I think we should log the Exception as well e.g. f"Failed to apply migrations due to: {e} Trying to continue...".

Reply:
Sure! Though in this case I've already thought about this :sweat_smile:

This is handled automatically by `logging.exception`, along with nice benefits like stack trace:

![image](https://github.com/user-attachments/assets/0785b034-f66a-44f9-884e-7134ae789204)